src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs

```diff
@@ -59,39 +59,35 @@ internal static class ClassificationHelpers
     }
 
     private static bool IsControlKeyword(SyntaxToken token)
-    {
-        if (token.Parent is null || !IsControlKeywordKind(token.Kind()))
-        {
-            return false;
-        }
-
-        return IsControlStatementKind(token.Parent.Kind());
-    }
+        => token.Parent is not null &&
+            IsControlKeywordKind(token.Kind()) &&
+            IsControlStatementKind(token.Parent.Kind());
 
     private static bool IsControlKeywordKind(SyntaxKind kind)
     {
         switch (kind)
         {
-            case SyntaxKind.IfKeyword:
-            case SyntaxKind.ElseKeyword:
-            case SyntaxKind.WhileKeyword:
-            case SyntaxKind.ForKeyword:
-            case SyntaxKind.ForEachKeyword:
-            case SyntaxKind.DoKeyword:
-            case SyntaxKind.SwitchKeyword:
+            case SyntaxKind.AwaitKeyword:
+            case SyntaxKind.BreakKeyword:
```

Comment:
sorted just for clarity.  the only other hcange is adding the .AwaitKeyword entry.